src/Action.java

```diff
@@ -0,0 +1,5 @@
+public interface Action{
+	public String title = "untitled action";
```

Comment:
т.к. в интерфейсах поля по дефолту static И final, то писать их надо в верхнем регистре, т.е.
```suggestion
	public String TITLE = "untitled action";
```